src/test/java/org/prebid/server/bidder/adoppler/AdopplerBidderTest.java

```diff
@@ -84,7 +84,7 @@ public void makeHttpRequestsShouldCreateCorrectURL() {
         // then
         assertThat(result.getErrors()).isEmpty();
         assertThat(result.getValue()).hasSize(1);
-        assertThat(result.getValue().get(0).getUri()).isEqualTo(ENDPOINT_URL);
+        assertThat(result.getValue().get(0).getUri()).isEqualTo(ENDPOINT_URL+"/processHeaderBid/adUnit");
```

Comment:
style
+ cant you just add `/processHeaderBid/adUnit` to ENDPOINT_URL ?

Reply:
That will be incorrect